sections/expansion_content.tex

```diff
@@ -51,66 +75,189 @@ \subsection*{Empowered Statistic Cards}
   \medskip
 
   \begin{minipage}[h]{\linewidth}
-    \vspace{0.1pt}
-    \centering
-    \begin{scriptsize}
-      \begin{tikzpicture}
-        \draw (0, 0) node[inner sep=0] {\makebox[\linewidth][c]{\includegraphics[width=\linewidth]{\cards/event.png}}};
-        \draw (1, 1.8) node {\encircle{\phantom{.}1\phantom{.}}};
-        \draw (-2, 0.4) node {\encircle{\phantom{.}2\phantom{.}}};
-        \draw (2.5, -1.1) node {\encircle{\phantom{.}3\phantom{.}}};
-      \end{tikzpicture}
-    \end{scriptsize}
-    \footnotesize
-    \textbf{\textit{\textcolor{darkcandyapplered}{Event Card}}}
-    \begin{multicols}{2}
-      \begin{itemize}
-        \item[\textbf{1.}] Name
-        \item[\textbf{2.}] Fluff
-        \item[\textbf{3.}] Effect
-        \item[\textbf{\phantom{.}}] \phantom{.}
-      \end{itemize}
-    \end{multicols}
+  \vspace{0.1pt}
+  \centering
+  \begin{scriptsize}
+    \begin{tikzpicture}
+    \draw (0, 0) node[inner sep=0] {\makebox[\linewidth][c]{\includegraphics[width=\linewidth]{\cards/event.png}}};
+    \draw (1, 1.8) node {\encircle{\phantom{.}1\phantom{.}}};
+    \draw (-2, 0.4) node {\encircle{\phantom{.}2\phantom{.}}};
+    \draw (2.5, -1.1) node {\encircle{\phantom{.}3\phantom{.}}};
+    \end{tikzpicture}
+  \end{scriptsize}
+  \footnotesize
+  \textbf{\textit{\textcolor{darkcandyapplered}{Event Card}}}
+  \begin{multicols}{2}
+    \begin{itemize}
+    \item[\textbf{1.}] Name
+    \item[\textbf{2.}] Fluff
+    \item[\textbf{3.}] Effect
+    \item[\textbf{\phantom{.}}] \phantom{.}
+    \end{itemize}
+  \end{multicols}
   \end{minipage}
 \end{expansion}
 
-\end{multicols*}
+\columnbreak
+\begin{expansion}[title=]{rampart}
+  \subsection*{\color{rampart}Rampart Expansion}
+  \setlength\intextsep{0pt}
+  \setlength\columnsep{0.8em}
+  \begin{wrapfigure}{l}{0.35\textwidth}
+    \includegraphics[width=\linewidth]{\boxcovers/rampart-box.png}
+  \end{wrapfigure}
+  Contains the \textbf{Rampart Faction}: nature-oriented Units under the lead of Druids and Rangers.\par
```

Comment:
For future's sake, for simple edits like that, please use suggestions:
<img width="752" height="300" alt="Screenshot From 2025-10-07 21-30-18" src="https://github.com/user-attachments/assets/49e05d05-44f6-4530-a332-0ba1288a5bfb" />
```suggestion
  Contains the \textbf{Rampart Faction}: Nature-oriented Units under the lead of Druids and Rangers.\par
```